examples/DDG4/CMakeLists.txt

```diff
@@ -9,11 +9,16 @@
 #
 #==========================================================================
 cmake_minimum_required(VERSION 3.12 FATAL_ERROR)
-include ( ${DD4hep_DIR}/cmake/DD4hep.cmake )
 
+IF(NOT TARGET DD4hep::DDCore)
+  find_package ( DD4hep REQUIRED )
+  include ( ${DD4hep_DIR}/cmake/DD4hep.cmake )
+  include ( ${DD4hep_DIR}/cmake/DD4hepBuild.cmake )
+ENDIF()
+
+dd4hep_configure_output()
```

Comment:
Move this into the `IF` nad replace below
```
OUTPUT ${LIBRARY_OUTPUT_PATH}
```

So that all these blocks are the same in all examples?